### DISCUSSION TITLE
[docs] How to generate in the fly big component in QVirtualScroll

### DISCUSSION THREAD

**Discussion by MaYoRr** [quasarframework/quasar#10100] [General - Components / Directives / etc]
I have a large self-written component for chat, the data I take from the backend. How do I make a query when there are less than 7 elements to the end of the array? The example from the documentation doesn't work  

![image](https://user-images.githubusercontent.com/49597310/125988135-41b98fdb-383d-4215-9a91-544acc3711f8.png)

**Comment by pdanpdan** [comment]
What do you mean by the example does not work?
Make a codepen with what you are doing.

**Reply by ontwikkelfabriek** [reply]
@MaYoRr 

I guess you solved it?
